master/getting-started/kubernetes/installation/hosted/kubernetes-datastore/index.md

```diff
@@ -2,28 +2,27 @@
 title: Kubernetes Datastore
 ---
 
-This document describes how to install {{site.prodname}} on Kubernetes in a mode that does not require access to an etcd cluster.
-This mode uses the Kubernetes API as the datastore.
+This document describes how to install {{site.prodname}} on Kubernetes without a separate etcd cluster.
+In this mode, {{site.prodname}} uses the Kubernetes API directly as the datastore.
 
-Note that this feature currently comes with a number of limitations, namely:
+Note that this mode currently comes with a number of limitations, namely:
 
 - It does not yet support Calico IPAM.  It is recommended to use `host-local` IPAM in conjunction with Kubernetes pod CIDR assignments.
-- It does not yet support per-node low-level Felix configuration. This must be handled using the Felix environment variables
-  passed into the `calico/node` container (see [Configuring Felix]({{site.baseurl}}/{{page.version}}/reference/felix/configuration)).
 - {{site.prodname}} networking support is in beta. Control of the node-to-node mesh, default AS Number and all BGP peering configuration should be configured using `calicoctl`.
```

Comment:
So here, it still says that Calico networking is in beta. But elsewhere, I noticed that you removed this.

Reply:
I think we need to keep calico networking tagged as "beta" until we've implemented Calico IPAM for KDD mode, which we haven't yet.